modules/ROOT/pages/afm-monitoring-public-apis.adoc

```diff
@@ -42,14 +66,10 @@ To prevent tests run in public locations from consuming too many resources, AFM
 
 Tests are run at fixed intervals that you define in schedules. Each test is associated with a schedule. When you run tests against the endpoints of public APIs, AFM sets limitations on the following aspects of scheduling:
 
-Number of Schedules::
-+
-By default, AFM lets your organization in Anypoint Platform run up to five schedules for testing from public locations at one time. If you reach the limit of five schedules and want to run more monitors simultaneously, contact your MuleSoft Customer Success Manager about raising your maximum.
+* *Number of Schedules*: By default, your organization can run up to five schedules at one time for testing from public locations. If you reach the limit of five schedules and want to run more monitors simultaneously, contact your MuleSoft Customer Success Manager about raising your maximum.
 
-You can also run monitors from private locations, if you have access to Anypoint Virtual Private Cloud. There is no limit to the number of schedules that you can create to use private locations.
+* *Duration of Intervals Between Executions of Tests*: Fifteen minutes is the shortest interval that you can schedule to occur between tests.
```

Comment:
Why repeat "to occur between tests" (it's all in the name of the config variable)? I'd break it into two sentences or drop the final prepositional phrase for readability?

Reply:
Yeah, that's a good call. I'll cut it.